README.md

```diff
@@ -110,6 +110,20 @@ latest 2 versions of an OS.
 | Android NDK | 1 latest non-LTS, 2 latest LTS versions |
 | Xcode     | - all OS compatible versions side-by-side <br/> - for beta, GM versions - latest beta only <br/> - old patch versions are deprecated in 3 months |
 
+### Package managers usage
+
+We use a number of third-party package managers during image generation process for software installation.
+
+The table below reflects the list of these package managers and the software installed.
+
+
+| Operating system | Package manager | Third-party repos and packages |
+| :---        |     :---:      |          ---: |
+| Ubuntu      | [APT](https://wiki.debian.org/Apt) <br/> [pipx](https://pypa.github.io/pipx)        | firefox - `mozillateam/ppa` <br> git - `git-core/ppa` <br/> php - `ondrej/php` (`ubuntu-18.04` and `ubuntu-20.04` only) <br/> gcc, gfortran -  `ubuntu-toolchain-r/test` <br/><br/>ansible-core, yamllint - `pipx` <br/><br/> APT repositories: <br/><br/> [Eclipse-Timurin (Adoptium)](https://packages.adoptium.net/artifactory/deb) <br/> [Erlang](https://packages.erlang-solutions.com/ubuntu) <br/> [HHvm](https://dl.hhvm.com/ubuntu) <br/> [Mono](https://download.mono-project.com/repo/ubuntu) <br/> [PostgreSQL](https://apt.postgresql.org/pub/repos/apt) </br> [R](https://cloud.r-project.org/bin/linux/ubuntu)|
+| Windows     | [Chocolatey](https://chocolatey.org)  | No third party reposetories used     |

```

Comment:
for the middle column, can we put package managers on its own row and in the right column separate out the tools based on the package manager we use? you'll have to use html in this case since it's a bit advanced for markdown but i think would look cleaner (since you're trying to distinguish them anyway by the suffix at the end)